maps/tether/tether_defines.dm

```diff
@@ -24,6 +24,7 @@
 #define Z_LEVEL_DEBRISFIELD					23
 #define Z_LEVEL_FUELDEPOT					24
 #define Z_LEVEL_GATEWAY						25
+#define Z_LEVEL_GUTTERSITE					26
```

Comment:
This doesn't seem right. You're loading it BEFORE the fuel depot. It's probably not z 26. Can you ghost over your map ingame and verify what zlevel it is?

Reply:
The load order *does* matter, I found out - seconding this